applications/directory/src/components/CreateAccount.js

```diff
@@ -63,31 +130,66 @@ function CreateAccount() {
             }}
             sx={{ minWidth: ['100%', '100%', '325px'] }}
             required
+            onChange={handleChange}
+            onBlur={async (e) => {
```

Comment:
I would move all this to a handler...like `handleEmailBlur()`.

Reply:
done